src/main/java/edu/byu/cs/util/SubmissionUtils.java

```diff
@@ -75,8 +75,7 @@ public static void approveSubmission(
         // Send score to Grade-book
         float approvedScore = Scorer.prepareModifiedScore(withheldSubmission.score(), penaltyPct);
         String gitCommitsComment = "Submission initially blocked due to low commits. Submission approved by admin " + approverNetId;
-        Scorer.attemptSendToCanvas(withheldSubmission.rubric(), withheldSubmission.phase(), withheldSubmission.netId(),
-                penaltyPct, gitCommitsComment);
+        Scorer.attemptSendApprovedScoreToCanvas(withheldSubmission, penaltyPct, gitCommitsComment);
```

Comment:
Acknowledging the current model with `Submission`s having original scores without late penalties applied, this now becomes the correct place to apply the late penalties. The Scorer methods most reasonably assume that all calculations have already occurred. Doing it lower exposes the option of double-penalizing, besides the previously mentioned issues with dependencies.